src/services/balancer/subgraph/entities/pools/index.ts

```diff
@@ -10,6 +10,13 @@ import {
 } from '../../types';
 import { Prices } from '@/services/coingecko';
 import { getAddress } from '@ethersproject/address';
+import {
+  currentLiquidityMiningRewards,
+  computeAPYForPool
+} from '@/lib/utils/liquidityMining';
+import { TOKENS } from '@/constants/tokens';
+
+const IS_LIQUIDITY_MINING_ENABLED = true;
```

Comment:
@garethfuller once the PR is reviewed and approved - I'll set it to this value to `false` so that it won't block any releases until we have the correct JSON file in place.